fe2-android/app/src/test/ui/robolectric/com/github/dedis/popstellar/ui/lao/event/rollcall/RollCallArrayAdapterTest.kt

```diff
@@ -0,0 +1,67 @@
+package com.github.dedis.popstellar.ui.lao.event.rollcall
+
+import android.content.Context
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import androidx.test.core.app.ApplicationProvider
+import com.github.dedis.popstellar.R
+import com.github.dedis.popstellar.model.objects.security.PoPToken
+import net.i2p.crypto.eddsa.Utils
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.Mockito.mock
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class RollCallArrayAdapterTest {
+
+    @Mock
+    private lateinit var mockView: View
+
+    private lateinit var adapter: RollCallArrayAdapter
+
+    val context = ApplicationProvider.getApplicationContext<Context>()
+
+    private val MY_PRIVATE_KEY =
+            Utils.hexToBytes("3b28b4ab2fe355a13d7b24f90816ff0676f7978bf462fc84f1d5d948b119ec66")
```

Comment:
It's fine for me having this hardcoded, as it doesn't impact the test coverage, but we have in the codebase a random generator for keys that we use in test, just pointing this for future work as it's gonna save you some time

Reply:
Oh ok, thanks for the information